components/list/demo/demo-list-nested-iterations-helper.js

```diff
@@ -0,0 +1,126 @@
+import '../list-item-content.js';
+import '../list-item.js';
+import '../list.js';
+import { css, html, LitElement, nothing } from 'lit';
+import { tableStyles } from '../../table/table-wrapper.js';
+
+const styles = css`
+	:host {
+		display: block;
+	}
+	d2l-list:not([slot="nested"]) {
+		border: solid 1px black;
+		margin: 1rem;
+		padding: 1rem;
+	}
+	d2l-list:not([slot="nested"]) > d2l-list-item > d2l-list-item-content {
+		color: blue;
+	}
+	d2l-list-item {
+		box-shadow: -1px 0 0 0 red;
+	}
+	.minimize-width {
+		width: 1.5rem;
+	}
+`;
+
+class ListNestedIterationsHelper extends LitElement {
+	static get properties() {
+		return {
+			draggable: { type: Boolean }
+		};
+	}
+
+	static get styles() {
+		return [tableStyles, styles];
+	}
+
+	constructor() {
+		super();
+		this.draggable = false;
+	}
+
+	render() {
+		const selectableOptions = [
+			{ name: 'All are Selectable', parent: true, child: true },
+			{ name: 'None are Selectable', parent: false, child: false },
+			{ name: 'Children Selectable (Parent Not)', parent: false, child: true },
+			{ name: 'Parent Selectable (Children Not)', parent: true, child: false }
+		];
+
+		const tableRows = selectableOptions.map(option => html`
+			<tr class="d2l-table-header">
+				<th rowspan="2" scope="rowgroup">${option.name}</th>
+				<th scope="row">Exp/Collapsible Children</th>
+				<td>${this._createList([option.parent, true], [option.child, true])}</td>
+				<td>${this._createList([option.parent, false], [option.child, true])}</td>
+			</tr>
+			<tr class="d2l-table-header">
+				<th scope="row">Non-Exp/Collapsible Children</th>
+				<td>${this._createList([option.parent, true], [option.child, false])}</td>
+				<td>${this._createList([option.parent, false], [option.child, false])}</td>
+			</tr>
+		`);
+
+		return html`
+			<d2l-table-wrapper>
```

Comment:
I'm thinking it might be better to _not_ use our table, to avoid these visual-diffs changing when we make table changes. Used them to get things started since they looked nice, but I could try to copy a subset of the styles. Thoughts?

Reply:
I could also offer a non-table version that just returns them all with spacing for visual-diff only

Reply:
Yeah, I think my vote would be to isolate from other components.

Reply:
Specifically for visual-diff.

Reply:
+1. Seems like you could accomplish what you need with a native `<table>` and some simple CSS.